### ISSUE TITLE
algo  timetravel10 something the matter

### ISSUE THREAD

**Issue by jocking** [JayDDee/cpuminer-opt#25]
cpuminer-opt 3.6.7 use timetravel10 algo  
cpuminer-aes-avx2.exe has stopped working and a problem has occurred causing the program to stop working.
What's the problem, please? How can it be solved?

**Comment by JayDDee** [comment]
You're doing something wrong. This is not the place for noobs that don't know what the're doing.
Learn how to run console programs and read the help.

**Comment by jocking** [comment]
Thank you. No operation error. I have read the help.
I have been testing other algorithms, such as qubit, X11, scrypt etc. there is no problem, but timetravel10 is not working, I operation way, **cpuminer-aes-avx2.exe -a timetravel10 -o stratum+tcp://btx.suprnova.cc:3629 -u btx1.btx -p 1 -t 2**
Could you help me see where it's wrong?

**Comment by aaronsace** [comment]
Confirmed, bitcore is not working. 

Output from console: 

```
[2017-07-26 02:55:24] Starting Stratum on stratum+tcp://bitcore.mine.zpool.ca:3556
[2017-07-26 02:55:24] 8 miner threads started, using 'timetravel10' algorithm.
[2017-07-26 02:55:24] Stratum difficulty set to 0.5
[2017-07-26 02:55:30] timetravel10 block 19922, diff 303.732
```

**Comment by jocking** [comment]
Thank you !

Yes. Minne too . Output from console
[2017-07-26 10:04:32] 2 miner threads started, using 'timetravel10' algorithm.
[2017-07-26 10:04:32] Starting Stratum on stratum+tcp://bitcore.mine.zpool.ca:3556
[2017-07-26 10:04:35] Stratum difficulty set to 0.5
[2017-07-26 10:04:59] timetravel10 block 19924, diff 303.732
 Press any key to continue

A pleasant exchange! Thank you for your open source. Although I'm not proficient in C

**Comment by JayDDee** [comment]
Thanks for the additional info. I've stopped asking for proper problem descriptions and just
dismiss poorly defined issues. It's still not enough to meet minimum standards but enough to get
my attention. 

I have confirmed the Windows binaries don't work mining timretravel10 in any release since the
algo was introduced. It works on Linux. I will investigate.

**Comment by JayDDee** [comment]
Fixed in v3.6.8
